src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php

```diff
@@ -100,9 +100,26 @@ public function getHitCount(): int
         return (int) floor($this->hitCountForLastWindow * (1 - $percentOfCurrentTimeFrame) + $this->hitCount);
     }
 
-    public function getRetryAfter(): \DateTimeImmutable
```

Comment:
You cannot remove public methods. That is a BC break.

Reply:
My bad, thought the `@internal` class allowed this.